src/main/java/duke/gui/MainWindow.java

```diff
@@ -559,6 +559,12 @@ public void updateTableViews() {
         patientSearchBarListener();
         assignedTaskSearchBarListener();
         taskSearchBarListener();
+
+        try {
```

Comment:
Same for this DukeException.

Reply:
I think you may have been referring to the other altered exception in the CommandManager class? This particular try/catch didn't exist until I added it in last night.

Reply:
Good!